src/main/java/com/cricket/project/controller/PlayerController.java

```diff
@@ -0,0 +1,47 @@
+package com.cricket.project.controller;
+
+import com.cricket.project.model.Player;
+import com.cricket.project.model.PlayerScoreCard;
+import com.cricket.project.service.impl.PlayerServiceImpl;
+import com.cricket.project.service.impl.ScoreCardServiceImpl;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@Slf4j
+public class PlayerController {
+    @Autowired
+    private PlayerServiceImpl playerService;
+    @Autowired
+    private ScoreCardServiceImpl scoreCardService;
+
+    @PostMapping("/addPlayer")
+    public String addPlayer(@RequestBody Player player) {
+        return playerService.addPlayer(player);
+    }
+
+    @GetMapping("/players")
```

Comment:
we can use `players` as the base route and to return all the players we can use `/players/all`